tests/jballerina-unit-test/src/test/resources/test-src/query/multiple-let-clauses.bal

```diff
@@ -162,6 +162,30 @@ function testQueryExpressionWithinLetClause() {
     assertEquality(1, emp.deptId);
 }
 
+public function testwildcardBindingPatternInLetClause() {
+
+    Person p1 = {deptAccess: "OP", firstName: "Ranjan", lastName: "Fonseka"};
+    Person p2 = {deptAccess: "MKT", firstName: "Mark", lastName: "George"};
+    Person p3 = {deptAccess: "ENG", firstName: "Grainier", lastName: "Perera"};
+    Person[] personList = [p1, p2, p3];
+    Person[] outputPersonList = from var person in personList
```

Comment:
Shall we try an error binding in the let. Gave me a npe with the following sample.

```ballerina
function bar() returns error {
    return error("", message = "asdasd");
}

Person[] outputPersonList = from var person in personList
    let error error(_, message = message1) = bar()
    let var [_, name] = foo()
    where person.deptAccess == "MKT" && person.firstName == name
    select {
        firstName: name,
        lastName: person.lastName,
        deptAccess: "WSO2"
    };

```